lib/rubocop/cop/cop.rb

```diff
@@ -142,10 +148,15 @@ def find_location(node, loc)
         loc.is_a?(Symbol) ? node.loc.public_send(loc) : loc
       end
 
+      # TODO: remove - it's unused
```

Comment:
`Cop::Cop` is being heavily refactored, all these changes will conflict, but thanks for reminding me to optimize `duplicate_location?` as [`Source::Range` can now be used as hash keys / set elements](https://github.com/whitequark/parser/pull/675)

Reply:
@marcandre Would you mind my opening a separate PR with this optimization if it's a straightforward solution?